notebooks/WET/WET-003/WET-003-ComCam.ipynb

```diff
@@ -7,217 +7,757 @@
    "source": [
```

Comment:
<p>But you wrote on slack that Times Square cannot read donut stamps - so I can't really see that on Times Square, right?  (I  can confirm that the plots show up when running the notebook at USDF)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>